code/modules/food_and_drinks/food/foods/candy.dm

```diff
@@ -644,3 +644,16 @@
 	icon_state = "toolerone"
 	filling_color = "#7D5F46"
 	goal_difficulty = FOOD_GOAL_NORMAL
+
+// ***********************************************************
+// OmniForge Solutions - Candy
+// ***********************************************************
+
+/obj/item/food/candy/chocolate_bar
+	name = "Chocolate bar - OFS"
+	desc = "Crafted from premium cacao and infused with quantum nutrients, this luxury chocolate bar is wrapped in nanofoil for ultimate freshness. Experience the essence of culinary supremacy with every bite."
+	icon_state = "choco_OFS"
+	filling_color = "#7D5F46"
+	list_reagents = list("chocolate" = 5, "sugar" = 10)
+	tastes = list("chocolate" = 3)
```

Comment:
why ```chocolate = 3``` if its the only taste it can have?

Reply:
To be fair i was not sure what the number for the taste variable was for , if it was the amount of tastes then I can lower it to 1 if that make sense.